website_sale_wishlist/models/res_users.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from openerp import api, fields, models
+from openerp.http import request
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    current_session = fields.Char(compute="_compute_current_session")
+
+    @api.one
```

Comment:
[There's still a little hope :pray: ](https://github.com/odoo/odoo/pull/13199). I don't consider it deprecated until that PR is closed.